charts/camunda-platform/templates/zeebe/statefulset.yaml

```diff
@@ -85,6 +85,35 @@ spec:
               value: {{ .Values.zeebe.ioThreadCount  | quote }}
             - name: ZEEBE_BROKER_GATEWAY_ENABLE
               value: "false"
+            {{- if .Values.global.elasticsearch.external  }}
+              {{- if .Values.global.elasticsearch.openSearch}}
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_AUTHENTICATION_USERNAME
+              value: {{ .Values.global.elasticsearch.auth.username | quote }}
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_AUTHENTICATION_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: elastic-password
+                  key: password
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_CLASSNAME
+              value: "io.camunda.zeebe.exporter.opensearch.OpensearchExporter"
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_URL
+              value: {{ include "camundaPlatform.elasticsearchURL" . | quote }}
```

Comment:
💭 @nicpuppa and I wonder if it would be worth separating opensearch and elasticsearch properties completely otherwise it seems a bit confusing to configure elasticsearch properties to be used for opensearch setup?